src/oc_erchef/apps/oc_chef_wm/src/chef_reindex.erl

```diff
@@ -235,9 +235,10 @@ send_to_index_queue(OrgName, OrgId, Index, SerializedObjects, NameIdDict) ->
     {Res, Missing}.
 
 add_batch(Batch, OrgName) ->
-    case oc_chef_object_db:add_batch_to_solr(Batch) of
-        ok -> ok;
-        {error, Failures} ->
+    try oc_chef_object_db:add_batch_to_solr(Batch) of
+        ok -> ok
+    catch
+        error:Failures ->
```

Comment:
I don't think the error:Failures catch will catch an `{error, Failures}` tuple returned from add_batch_to_solr.